### PR TITLE
{tools}[GCC/11.2.0] example v1.0.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

please merge my super software

### CODE REVIEW

File: easybuild/easyconfigs/e/example/example-1.0.1-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'https://easybuilders.github.io/easybuild-tutorial'
+description = "EasyBuild tutorial example"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/easybuilders/easybuild-tutorial/raw/main/docs/files/']
+sources = ['eb-tutorial-%(version)s.tar.gz']
+checksums = ['d6cec2ea298f4092cb1b880cb017220ab191561da941e9e480639cf3354b7ef9']
+
+builddependencies = [('CMake', '3.22.1')]
```
Comment (boegel) [inline easybuild/easyconfigs/e/example/example-1.0.1-GCC-11.2.0.eb:15]: Let's format this:

```python
builddependencies = [
    ('CMake', '3.22.1'),
]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by bartoldeman** [comment]
@boegelbot please test @ generoso

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/akesandgren/38bffc44dbdd09b9f345ebb39d12e5b5 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
eb-mix.zao.se - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, AMD Ryzen 9 3900X 12-Core Processor (zen2), Python 3.9.14
See https://gist.github.com/zao/c12273b0dd11ec0b63914e6172f71728 for a full test report.

**Pull request closed without merging**